chain/bitcoind_client.go

```diff
@@ -599,10 +599,11 @@ func (c *BitcoindClient) ntfnHandler() {
 	for {
 		select {
 		case tx := <-c.txNtfns:
-			_, _, err := c.filterTx(tx, nil, true)
+			txHash := tx.TxHash()
```

Comment:
Alternatively, we can use `btcutil.Tx` as a wrapper here which will actually cache the hash by default at all times. Either way gives us the desired end result tho.

Reply:
Ah, wasn't aware of that. Sounds good, updated.